sycl/unittests/pi/cuda/test_primary_context.cpp

```diff
@@ -0,0 +1,134 @@
+//==---------- pi_primary_context.cpp - PI unit tests ----------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
Could you add a TODO here, please?
It looks like we need to do something to enable these tests. Right?

Reply:
Indeed, we should do that after we have stable LIT results.